.eslintrc.yml

```diff
@@ -108,3 +111,12 @@ overrides:
         # disallow Reporters using `console.log()`
         - selector: 'CallExpression[callee.object.name=console][callee.property.name=log]'
           message: &GH-3604 See https://github.com/mochajs/mocha/issues/3604
+  - files:
+      - browser-entry.js
+      - lib/**/*.js
+    excludedFiles:
+      - lib/cli/**/*.js
+      - lib/browser/growl.js
```

Comment:
You can exclude these files. 

```
- lib/esm-utils.js
- lib/nodejs/serializer.js
- lib/nodejs/parallel-buffered-runner.js
- lib/nodejs/reporters/parallel-buffered.js
- lib/nodejs/worker.js
- lib/nodejs/buffered-worker-pool.js
```

Reply:
Thank you for your guidance and kindness. I take this.